WalletWasabi/Discoverability/NostrCoordinatorPublisher.cs

```diff
@@ -0,0 +1,44 @@
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using NBitcoin;
+using NBitcoin.Secp256k1;
+using NNostr.Client;
+using WalletWasabi.Bases;
+using WalletWasabi.Logging;
+
+namespace WalletWasabi.Discoverability;
+
+public class NostrCoordinatorPublisher : PeriodicRunner
+{
+	public NostrCoordinatorPublisher(TimeSpan period, AnnouncerConfig config, Network network) : base(period)
+	{
+		Key = ECPrivKey.Create(config.KeyBytes);
+		CoordinatorConfiguration = new NostrCoordinatorConfiguration(config.CoordinatorDescription, new Uri(config.CoordinatorUri), network);
+		Client = NostrExtensions.Create(config.RelayUris.Select(x => new Uri(x)).ToArray(), (EndPoint?)null);
```

Comment:
No. The proxy has to be passed. You shouldn't pass `null` here.

Reply:
What is the proxy that should be passed?

Reply:
This will be done post merge when PR about replacing our custom HttpClient Tor Proxy implementation with WebProxy will be merged